drivers/media/i2c/imx296.c

```diff
@@ -171,6 +171,10 @@
 #define IMX296_GTTABLENUM				IMX296_REG_8BIT(0x4114)
 #define IMX296_CTRL418C					IMX296_REG_8BIT(0x418c)
 
+#define IMX296_STANDBY_DELAY		1500
+#define IMX296_STREAM_ON_DELAY		2000
+#define IMX296_STREAM_OFF_DELAY		2000
```

Comment:
Where have these delays come from? The only numbers I can see are in "Sensor Setting Flow (Sensor Master Mode)" for a >1ms delay "Wait for internal regulator stabilization".